### PR TITLE
Search

### PR DESCRIPTION
**Pull request by jonathanwatters**
Patch (& description) search functionality

NEEDS WORK - UISearchBar sizing

### CODE REVIEW

File: Podfile.lock [hunk 1]
```diff
@@ -46,4 +46,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 5a6fc3652ff27185dbe549b41dce70a850fa2ea3
 
-COCOAPODS: 1.7.0
+COCOAPODS: 1.6.1
```
Comment (sieren) [inline Podfile.lock]: This probably shouldnt be in this PR

**Comment by sieren** [comment]
Looks good on first look, @marcussatellite are you gonna test this locally?

**Comment by jonathanwatters** [comment]
>Looks good on first look, @marcussatellite are you gonna test this locally?

@sieren My problem is that it doesn’t *look* good - the UISearchBar is extending where it shouldn’t. Need a bit of advice on that. 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

File: Podfile.lock (hunk 1 above)
Comment (jonathanwatters) [reply Podfile.lock]: I now have cocoapods 1.7.0 installed, & pushed revised lockfile which has caused it to drop out of this PR

**Pull request merged**